.github/pull_request_template.md

```diff
@@ -1 +1,46 @@
-Test
+*Put this in the title* \[W/B/T]\[PFXXX-XXXX\] \<PR Title\>
+
+Please make sure that you incorporate these in your pull request:
+- [ ] Summary of the changes you made
+- [ ] Link to Trello backlog
+- [ ] Link to the issue you are resolving
+- [ ] *(Optional)* Screenshot of the UI
+- [ ] *(Optional)* Screenshot of the testing result
+
+***
+### Description
+A few sentences describing the overall goals of the pull request's commits.
+
+### Related Pull Requests (PRs)
+List related PRs against other branches:
+
+branch | PR
+------ | ------
+other_pr_production | [link]()
+other_pr_master | [link]()
+
+### Todos
+- [ ] Tests
+- [ ] Documentation
+
+### Deploy Notes
```

Comment:
change deploy notes and steps to reproduce with
Screenshot / Video:
divided into
UI
Network
and tests